releases/v0.2.0.md

```diff
@@ -0,0 +1,75 @@
+# Release Notes for v0.2.0
+Confidential Containers has adopted a six-week release cadence.
+This is our first release on this schedule.
+This release mainly features incremental improvements to our build system and tests
+as well as minor features, adjustments, and cleanup.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+
+- Kata CI uses existing Kata tooling to build components.
+- Kata CI caches build environments for components.
+- Pod VM can be launched with measured boot. See [more info](https://github.com/confidential-containers/documentation/issues/40)
+- Incremental advances in signature support including verification of cosign-signed images.
+- KBS no longer required to use unencrypted images with SEV.
+- More rigorous versioning of sub-projects
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- Intel SGX
+- AMD SEV-ES
+- IBM Z SE
+
+The following platforms are in development:
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * S390x is not supported by the CoCo operator
```

Comment:
@fitzthum

I like to have the `s390x` vs. `s390` vs. IBM Z SE etc. being used more consistently. However, I see the rush for get the v0.2.0 release out.  Just an idea to discuss: It would be nice to create the release notes for the v0.3.0 right after the v0.2.0 is out, then we can go even with feature PRs updating the release notes as we go along the timeline. Wdyt?

Reply:
That's a good idea. I will try to remember to do this.

Reply:
Also let me at least make that S lowercase. I try to mention Z at the top as the Confidential Computing feature that is supported but then the technical stuff is supposed to say `s390x` to talk about the OS. These release notes were written by a few different people, though.